queries.sql

```diff
@@ -56,4 +56,21 @@ SELECT species.name, COUNT(animals.species_id) FROM animals JOIN species ON spec
 
 SELECT animals.name from animals JOIN owners ON owners.id = animals.owners_id JOIN species on species.id = animals.species_id WHERE animals.species_id = '2' AND animals.owners_id = '2';
 SELECT animals.name from animals JOIN owners ON owners.id = animals.owners_id WHERE animals.escape_attempts = '0' AND animals.owners_id = '5';
-SELECT full_name, COUNT(owners_id) FROM owners JOIN animals on owners.id = animals.owners_id GROUP BY full_name ORDER BY COUNT (owners_id) desc limit 1;
\ No newline at end of file
+SELECT full_name, COUNT(owners_id) FROM owners JOIN animals on owners.id = animals.owners_id GROUP BY full_name ORDER BY COUNT (owners_id) desc limit 1;
+
+
+----
+
+SELECT animals.name FROM animals JOIN visits ON animals.id = visits.animals_id WHERE vets_id = '1' ORDER BY visits.date_of_visit desc limit 1;
+SELECT COUNT(DISTINCT animals_id) FROM visits WHERE vets_id = '2';
+
+SELECT vets.name, specialization.vets_id, specialization.species_id FROM vets LEFT JOIN specialization ON specialization.vets_id = vets.id;
+SELECT animals.name , visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animals_id WHERE visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30' AND vets_id = '3';
```

Comment:
- [ ] This query should return all animals that visited Stephanie Mendez between April 1st and August 30th, 2020; and is two animals `Agumon` and ` Blossom` but your query return `Agumon` and `Ditto`. `Ditto` has never been visited by Stephanie Mendez, please kindly fix that 🙃 🙃

Reply:
![image](https://user-images.githubusercontent.com/69153321/208113250-b9bd5f84-bea1-4c8c-a218-bc3f745e5c8e.png)

Alright had to update my DB, But yeah now it returns blossom. 😄